src/components/hand-controls.js

```diff
@@ -24,18 +24,9 @@ module.exports.Component = registerComponent('hand-controls', {
     this.onTrackpadUp = function () { self.handleButton('trackpad', 'up'); };
     this.onTriggerDown = function () { self.handleButton('trigger', 'down'); };
     this.onTriggerUp = function () { self.handleButton('trigger', 'up'); };
-    this.addEventListeners();
```

Comment:
@dmarcos I don't think I removed anything else; the add/remove event listener methods were just rolled into pause() and play().